### PR TITLE
Support API 4.0.0

### PR DESCRIPTION
**Pull request by NhanAZ**
Tested and it works! Hope so...

### CODE REVIEW

File: plugin.yml [hunk 1]
```diff
@@ -12,13 +12,13 @@ version: 1.0.0
...
 website: https://github.com/KygekDev/Height
-api: 3.0.0
+api: 4.0.0
 main: KygekDev\Height\Height
 
 commands:
   height:
     description: Get player height location relative to sea level
-    usage: /height [player]
+    usage: /height <player>
```
Comment (KygekDev) [inline plugin.yml]: Why do you change this while the command accepts the Player parameter as optional?

File: src/KygekDev/Height/Height.php [hunk 2]
```diff
@@ -31,18 +31,15 @@ public function onCommand(CommandSender $sender, Command $command, string $label
         if ($command->getName() !== "height" || !$sender->hasPermission(self::COMMAND_PERMISSION)) return true;
 
         if (!isset($args[0])) {
-            if (!$sender instanceof Player) {
-                $this->sendMessage($sender, "Usage: /height <player>");
-                return true;
-            }
+            if (!$sender instanceof Player) return false;
```
Comment (KygekDev) [inline src/KygekDev/Height/Height.php:34]: What will it print if the command returns `false`?

Comment (NhanAZ) [reply src/KygekDev/Height/Height.php:34]: print `Usage: /height [player]`

> https://github.com/NhanAZ/Height/blob/main/plugin.yml#L21

File: plugin.yml (hunk 1 above)
Comment (NhanAZ) [reply plugin.yml]: Player is not an array
Try using the command /effect

Comment (KygekDev) [reply plugin.yml]: AFAIK, in command usage, `[]` means **optional argument**, while `<>` means **required argument**.

Comment (NhanAZ) [reply plugin.yml]: Here is another example
`commands.deop.usage=/deop <player>`
> https://github.com/pmmp/Language/blob/stable/eng.ini#L32

Comment (KygekDev) [reply plugin.yml]: > Here is another example `commands.deop.usage=/deop <player>`
> 
> > https://github.com/pmmp/Language/blob/stable/eng.ini#L32

That command requires the Player argument, but this plugin does not.

Comment (NhanAZ) [reply plugin.yml]: > AFAIK, in command usage, `[]` means **optional argument**, while `<>` means **required argument**.

Edited...

**Review by KygekDev — COMMENTED** [review]
[I CAN'T DELETE THIS COMMENT]

File: plugin.yml [hunk 3]
```diff
@@ -18,11 +18,11 @@ main: KygekDev\Height\Height
 commands:
   height:
     description: Get player height location relative to sea level
-    usage: /height <player>
+    usage: /height [player]
```
Comment (KygekDev) [inline plugin.yml:21]: Thanks.

File: src/KygekDev/Height/Height.php [hunk 4]
```diff
@@ -31,18 +31,15 @@ public function onCommand(CommandSender $sender, Command $command, string $label
...
 
             $height = $this->getHeight($sender);
             $this->sendMessage($sender, "You're " . ($height >= 0 ? $height . "m above" : abs($height) . "m below") . " sea level");
             return true;
         }
 
-        $player = $this->getServer()->getPlayer($args[0]);
-        if ($player === null) {
+        $player = $this->getServer()->getPlayerByPrefix($args[0]);
+        if (!$player) {
```
Comment (KygekDev) [inline src/KygekDev/Height/Height.php:42]: If `$player` is `null`, will `!$player` return `true`?

Comment (NhanAZ) [reply src/KygekDev/Height/Height.php:42]: Exactly!

Comment (NhanAZ) [reply src/KygekDev/Height/Height.php:42]: `!$player` Faster

Comment (KygekDev) [reply src/KygekDev/Height/Height.php:42]: You are correct, just knew you can do that in strict mode.

https://3v4l.org/ARdc6

Comment (NhanAZ) [reply src/KygekDev/Height/Height.php:42]: So... Does it need re-editing?

Comment (KygekDev) [reply src/KygekDev/Height/Height.php:42]: > So... Does it need re-editing?

No need

**Comment by KygekDev** [comment]
@NhanAZ Thanks for your contribution, I will merge this PR now

**Pull request merged**